mysql/toddl_test.go

```diff
@@ -36,7 +36,7 @@ func TestToSpannerType(t *testing.T) {
 			"b": schema.Column{Name: "b", Type: schema.Type{Name: "float"}},
 			"c": schema.Column{Name: "c", Type: schema.Type{Name: "tinyint", Mods: []int64{1}}},
 			"d": schema.Column{Name: "d", Type: schema.Type{Name: "varchar", Mods: []int64{6}}},
-			"e": schema.Column{Name: "e", Type: schema.Type{Name: "double"}},
+			"e": schema.Column{Name: "e", Type: schema.Type{Name: "numeric"}},
```

Comment:
This test didn't originally test numeric (it was based on postgres/toddl_test.go, but changed the numeric case to double).
Changing it back to numeric to test generation of Spanner NUMERIC.